packages/compiler/src/render3/partial/api.ts

```diff
@@ -177,7 +177,7 @@ export interface R3DeclareComponentMetadata extends R3DeclareDirectiveMetadata {
   changeDetection?: ChangeDetectionStrategy;
 
   /**
-   * An encapsulation policy for the template and CSS styles.
```

Comment:
here and in the other cases I really think "template" shouldn't really be used as it makes the whole argument more confusing (in my opinion at least).

I know that the shadow DOM does change the component's template, but I think the whole purpose of the encapsulation is to control the styles so I would try not to mention template where possible

Reply:
Yes, sorry I mentioned the shadow DOM my comment but of course the emulated mode also modifies the template...

My point here is that from a user point of view the main goal of the view encapsulation is to handle the styles, I feel like the template aspect comes in second place, sort of like a side effect of the styles' encapsulation...

But it is technically incorrect to only mentioning the styles... :thinking: 

I can definitely revert this :slightly_smiling_face:

Reply:
So I think that the best compromise here is to refer to "styling" which is a sort of abstract term for "all the implementation details that will result in the component having particular style when rendered".

```suggestion
An encapsulation policy for the component's styling.
```

Reply:
sure sounds great! :smile: